code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -119,95 +118,107 @@ There are several things that need to be remembered:
 		var/target_overlay = U.icon_state
 		if(U.adjusted == ALT_STYLE)
 			target_overlay = "[target_overlay]_d"
-		else if(U.adjusted == DIGITIGRADE_STYLE)
-			target_overlay = "[target_overlay]_l"
-
 
 		var/mutable_appearance/uniform_overlay
 
-		if(dna?.species.sexes)
-			if(body_type == FEMALE && U.female_sprite_flags != NO_FEMALE_UNIFORM)
-				uniform_overlay = U.build_worn_icon(default_layer = UNIFORM_LAYER, default_icon_file = 'icons/mob/clothing/under/default.dmi', isinhands = FALSE, femaleuniform = U.female_sprite_flags, override_state = target_overlay)
+		if(dna.species.sexes && (dna.species.bodytype & BODYTYPE_HUMANOID)) //Agggggggghhhhh
+			if(gender == FEMALE && U.adjusted != NO_FEMALE_UNIFORM)
+				uniform_overlay = U.build_worn_icon(default_layer = UNIFORM_LAYER, default_icon_file = 'icons/mob/clothing/under/default.dmi', isinhands = FALSE, femaleuniform = U.adjusted, override_state = target_overlay)
 
+		//Change check_adjustable_clothing.dm if you change this
+		var/handled_by_bodytype = TRUE
+		var/icon_file
 		if(!uniform_overlay)
-			uniform_overlay = U.build_worn_icon(default_layer = UNIFORM_LAYER, default_icon_file = 'icons/mob/clothing/under/default.dmi', isinhands = FALSE, override_state = target_overlay)
-
-		if(OFFSET_UNIFORM in dna.species.offset_features)
-			uniform_overlay.pixel_x += dna.species.offset_features[OFFSET_UNIFORM][1]
-			uniform_overlay.pixel_y += dna.species.offset_features[OFFSET_UNIFORM][2]
+			//I was told to leave support in for Digitigrade clothing incase someone wanted it. So, here it is.
+			//BEGIN SPECIES HANDLING
+			//if((dna?.species.bodytype & BODYTYPE_DIGITIGRADE) && (U.supports_variations & DIGITIGRADE_VARIATION))
+			//	icon_file = DIGITIGRADE_PATH
```

Comment:
Leave this commented, we already *have* digitigrade clothing in this code base - just not too many